mediachain/translation/simple/translator.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import unicode_literals
+from mediachain.translation.translator import Translator
+
+
+class SimpleTranslator(Translator):
+
+    @staticmethod
+    def translator_id():
+        return 'SimpleTranslator/0.1'
+
+    @staticmethod
+    def translate(parsed_metadata):
+        simple_json = parsed_metadata
+
+        # extract artist Entity
+        artist_name = simple_json['artist']
```

Comment:
So as you can see here, basically just copy & pasted the original code, then changed a couple lines to just directly copy in the metadata without remapping the keys at all. Idea is a simple pass-through translator.

Yeah, may be a good idea to add a few more changes other than just that one line.

Maybe detect if there's an artist key, and only build the MC artist entity if there is?

The exact way of getting the artist details may need to be adjusted again due to the new schema changes I made last night, and as we move toward a more finalized schema... E.g. many works in some of our sources list multiple creators. We can discuss it more in slack.